Rakefile

```diff
@@ -1,6 +1,18 @@
-require 'appraisal'
+require 'bundler/setup'
 require 'bundler/gem_tasks'
+require 'appraisal'
 require 'rspec/core/rake_task'
 
-RSpec::Core::RakeTask.new
-task :default => :spec
+
+RSpec::Core::RakeTask.new(:spec) do |spec|
+  spec.rspec_opts = ['--backtrace '] if ENV['DEBUG']
+  spec.verbose = true
+end
+
+desc 'Default: run the unit tests.'
+task default: [:all]
+
+desc 'Test the plugin under all supported versions.'
+task :all do |_t|
```

Comment:
This could probably be defined using appraisal's rake tasks as dependencies instead of using exec